### PR TITLE
Issue #SB-19773 fix: text replaced for no book available

### PR DESCRIPTION
**Pull request by subranil**

### CODE REVIEW

File: src/assets/i18n/en.json [hunk 1]
```diff
@@ -160,7 +160,7 @@
   "CREDITS_LICENCE_INFO": "Credits & License Info",
   "CSV_DOWNLOAD_SUCCESS": "{{%s}} saved in ‘Downloads’",
   "CURRENT_LANGUAGE": "Current Language ",
-  "REQUEST_CONTENT": "Request for Content",
+  "REQUEST_CONTENT": "LET US KNOW HERE",
```
Comment (swayangjit) [inline src/assets/i18n/en.json:163]: Remove upper case strings from en.json and add capitalization css in html.

File: src/app/resources/resources.component.scss [hunk 2]
```diff
@@ -54,10 +54,14 @@
   border-radius: 4px;
   background-color: #FFFFFF;
```
Comment (AmiableAnil) [inline src/app/resources/resources.component.scss:55]: use color from variable.css

File: src/app/resources/resources.component.scss [hunk 3]
```diff
@@ -54,10 +54,14 @@
...
   color: #004D9F;
```
Comment (AmiableAnil) [inline src/app/resources/resources.component.scss:56]: use the color from variable.css

File: src/app/resources/resources.component.scss [hunk 4]
```diff
@@ -54,10 +54,14 @@
...
+  padding-left: 16px;
+  padding-right: 16px;
 }
 .explore-more-books {
   --background: #024F9D;
```
Comment (AmiableAnil) [inline src/app/resources/resources.component.scss:61]: use the color from variable.css

Comment (AmiableAnil) [inline src/app/resources/resources.component.scss:61]: use the color from variable.css

File: src/app/resources/resources.component.scss [hunk 5]
```diff
@@ -44,25 +44,29 @@
...
   border-radius: 4px;
-  background-color: #FFFFFF;
+  background-color: $white;
   color: #004D9F;
+  padding-left: 16px;
+  padding-right: 16px;
 }
 .explore-more-books {
-  --background: #024F9D;
+  --background: #{$blue};
```
Comment (swayangjit) [inline src/app/resources/resources.component.scss:61]: There is an extra # in the  --background

**Pull request merged**